packages/@cdk-cosmos/core/test/feature/ecs-feature.test.ts

```diff
@@ -101,6 +112,27 @@ describe('ECS Feature', () => {
     expect(solarSystem2.ecs).toHaveResource('AWS::EC2::SecurityGroup', { SecurityGroupIngress: httpSgIngress });
   });
 
+  test('should be an have correct security groups with albInternalListenerCidr', () => {
+    expect(solarSystem3.ecs).toHaveResource('AWS::EC2::SecurityGroup', {
+      SecurityGroupIngress: arrayWith(
+        {
+          CidrIp: '10.0.0.1/24',
+          Description: 'from 10.0.0.1/24:8080',
+          FromPort: 8080,
+          IpProtocol: 'tcp',
+          ToPort: 8080,
+        },
+        {
+          CidrIp: '10.0.0.1/24',
+          Description: 'from 10.0.0.1/24:8443',
+          FromPort: 8443,
+          IpProtocol: 'tcp',
+          ToPort: 8443,
+        }
+      ),
+    });
+  });
+
   test('should match snapshot', () => {
     expect(solarSystemStack).toMatchSnapshot();
```

Comment:
Add shap shot here ?

Reply:
solarSystem3 is almost identical to solarSystem, except the port 8080 and 8443 changes. That changes are captured with line 115 to 135. Do I still need snapshot here?